articles/tutorials/04_working_with_textures/index.md

```diff
@@ -0,0 +1,381 @@
+> 2. Select *Properties* from the context menu.  This will open the *Properties* panel for the item.
+> 3. Change the *Copy to Output Directory* property to *Copy Always*.
+
+Now when you run the game, first the build is performed which will copy the image file to the build output directory, then the image is loaded as a texture in the `LoadContent` method using the `Texture2D.FromFile` method call.
+
+Now let's take a look at doing the same process but instead using the content pipeline.
+
+### Loading From Content Pipeline
+
+To use the content pipeline to load our image, we first need to open the *MonoGame Content Builder Editor (MGCB Editor)*. Since we are using VSCode for this tutorial, you should have installed the *MonoGame for VSCode* extension in [Chapter 02](../02_getting_started/index.md#installing-the-monogame-for-vscode-extension). With this extension installed, when you have a code file open, you will see the MonoGame logo in the top-right of the code window like below:
+
+<figure><img src="./images/mgcb-editor-icon.png" alt="Figure 4-3: MonoGame for VSCode extension icon"><figcaption><p><strong>Figure 4-3: MonoGame for VSCode extension icon</strong></p></figcaption></figure>
+
+Clicking this icon will open the MGCB Editor with the *Content.mgcb* file in the current project loaded.
+
+> [!NOTE]  
+> If you did not install the *MonoGame for VSCode* extension or prefer to not use it, you can use the CLI commands to open the MGCB Editor instead. To do this:
+>
+> 1. Open the terminal in VSCode pressing `` CTRL+` ``or choosing *View > Terminal* from the top menu
+> 2. If the terminal directory is not in the project root directory (the one with the .csproj file), then use the `cd` command to navigate to that directory.
+> 3. Enter the command `dotnet mgcb-editor ./Content/Content.mgcb` to open the MGCB Editor using the *Content.mgcb* file from the project.
+
+<figure><img src="./images/mgcb-editor.png" alt="Figure 4-4: MonoGame Content Builder Editor (MGCB Editor) Window"><figcaption><p><strong>Figure 4-4: MonoGame Content Builder Editor (MGCB Editor) Window</strong></p></figcaption></figure>
+
+With the MGCB Editor now open, perform the following
+
+1. Right-click on the *Content* node in the *Project* panel on the left.
+2. Select *Add > New Folder*.
+3. Name the folder `images`.
+4. Right-click on the new *images* folder
+5. Select *Add > Existing Item*.
+6. Navigate to the *icon.png* file and choose it
+
+After adding an existing file, you will be prompted with a pop-up asking if you would like to *Copy the file* or *Add a link*. Since the file is already in our *Content/images* directory, the copy option is greyed out, so choose the *Add a link* option, then click the *Add* button.
+
+<figure><img src="./images/add-file-popup.png" alt="Figure 4-5: Add Existing File Popup"><figcaption><p><strong>Figure 4-5: Add Existing File Popup</strong></p></figcaption></figure>

```

Comment:
Why does the image have the "copy" method greyed

Reply:
The image has the copy method greyed out as explained in the paragraph above.  Since the image file already exists within the `Content/images` directory, the MGCB Add File dialog has the option greyed out

Reply:
image has been updated to show all options with none greyed out